src/MTSerialization/OwnImplementation/maps/official.json

```diff
@@ -529,7 +529,7 @@
         },
         {
             "id": "-715532088",
-            "predicate": "userProfilePhoto",
+            "predicate": "userProfilePhoto_v1",
```

Comment:
```
$ git grep userProfilePhoto | grep predica
src/MTSerialization/OwnImplementation/maps/layer_105.json:            "predicate": "userProfilePhotoEmpty",
src/MTSerialization/OwnImplementation/maps/layer_105.json:            "predicate": "userProfilePhoto",
src/MTSerialization/OwnImplementation/maps/official.json:            "predicate": "userProfilePhoto_v1",
src/MTSerialization/OwnImplementation/maps/tg_app_old.json:            "predicate": "userProfilePhotos",
```
We need this to avoid multiple meanings for `userProfilePhoto` predicate.